shipyard-controller/main_test.go

```diff
@@ -845,8 +845,14 @@ func Test__main_SequenceStateParallelStages(t *testing.T) {
 	require.Equal(t, keptnv2.GetTriggeredEventType("delivery"), stage.LatestEvent.Type)
 
 	// get delivery.triggered event
-	deliveryTriggeredEvent := natsClient.getLatestEventOfType(*keptnContext.KeptnContext, projectName, "dev", keptnv2.GetTriggeredEventType("delivery"))
-	require.NotNil(t, deliveryTriggeredEvent)
+	var deliveryTriggeredEvent *apimodels.KeptnContextExtendedCE
```

Comment:
technically this change is out of scope for the PR, but this assertion proved to be flaky and occasionally fail